### PR TITLE
Component tests

### PR DESCRIPTION
**Pull request by samanpwbb**
Adds tests that ensure maki icons are compatible with style components. Tests work by converting all icons to the iconData format, then validating the iconData derived from each icon.

### CODE REVIEW

**Comment by samanpwbb** [comment]
@dasulit for review plz 🙏

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "prettier": "^1.14.3",
     "tap-spec": "^4.1.1",
```
Comment (dasulit) [inline package.json]: There's a security warning coming from lodash, which I think originates from this outdated package. If this is a quick fix, can you handle while you're in here?

File: test/maki.test.js [hunk 2]
```diff
@@ -142,3 +146,23 @@ test('valid svgs ', function(t) {
     );
   }
 });
+
+test('svg are compatible with style components', (t) => {
```
Comment (dasulit) [inline test/maki.test.js]: I'm concerned that this test will be opaque in intention to folks outside of the organization. Can we give more detail to the requirements here, so external contributors are able to troubleshoot?

**Comment by samanpwbb** [comment]
@dasulit added a clarifying comment. Back to you!

File: test/maki.test.js [hunk 3]
```diff
@@ -142,3 +146,31 @@ test('valid svgs ', function(t) {
...
   }
 });
+
+/*
+ * "Style components" are an experimental Mapbox feature that power Mapbox's
+ * internal map design workflow. Part of that workflow involves convert Maki
+ * SVGs into a JSON icon format that has some specific constraints. Maki icons
+ * should generally be compatible with style components after running the build
+ * script defined in package.json. If this test is failing for you, don't
+ * hesitate to reach out to github user @samanpwbb.
```
Comment (dasulit) [inline test/maki.test.js]: Maybe... `@samanpwbb or another Mapboxer` ? 🚌🚌🚌

**Pull request merged**